themes/src/main/resources/theme/base/account/messages/messages_ru.properties

```diff
@@ -0,0 +1,151 @@
+doSave=\u0421\u043E\u0445\u0440\u0430\u043D\u0438\u0442\u044C
+doCancel=\u041E\u0442\u043C\u0435\u043D\u0430
+doLogOutAllSessions=\u0412\u044B\u0439\u0442\u0438 \u0438\u0437 \u0432\u0441\u0435\u0445 \u0441\u0435\u0441\u0441\u0438\u0439
+doRemove=\u0423\u0434\u0430\u043B\u0438\u0442\u044C
+doAdd=\u0414\u043E\u0431\u0430\u0432\u0438\u0442\u044C
+doSignOut=\u0412\u044B\u0445\u043E\u0434
+
+editAccountHtmlTtile=\u0420\u0435\u0434\u0430\u043A\u0442\u0438\u0440\u043E\u0432\u0430\u0442\u044C \u0430\u043A\u043A\u0430\u0443\u043D\u0442
```

Comment:
In russian there's no word 'аккаунт', so sounds good to me.
The same thing with that 'realm' word, there's no analogue in russian, i will try to find something in literature during weekend, maybe, because i really don't like this only word being in english.